cluster/providers/ova/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+export NFS_IP_ADDRESS=$(ip route get 8.8.8.8 | awk '{ print $7 }' | head -1)
+export NFS_SHARE="/home/nfsshare"
+
+ova_file_url="https://github.com/kubev2v/forkliftci/releases/download/v8.0/vm.ova"
+
+# Download the file and check if the copy was successful
```

Comment:
The `mount` command can't be executed without using sudo (I'm not sure if its possible in th CI), switch to flow where we must create the local NFS share from the CI - using `install-provider.sh` with `INSTALL_NFS=true` , then directly coping the file the the shared local folder instead of mounting the share, by doing so the tests for OVA can't be locally run with different NFS share so it should be documented.
I case its possible to run sudo command from the CI, like we do for `install_nfs.sh` i"ll switch back to the older flow.

Reply:
it should be possible to use sudo if we use it elsewhere I believe

Reply:
ack, reverting back, if we see later on that the CI not passing we can switch to the second version.